bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/sdq/eclipse/common/core/GradingSystemwideController.java

```diff
@@ -317,16 +317,7 @@ public void submitAssessment() {
 	}
 
 	private void updateConfigFile() {
-		if (this.preferenceStore.getBoolean(PreferenceConstants.IS_RELATIVE_CONFIG_PATH)) {
-			if (this.course != null && this.exercise != null && this.submission != null) {
-				// not the case at startup with rel config path chosen!
-				this.setConfigFile(new File(ResourcesPlugin.getWorkspace().getRoot().getProject(this.getCurrentProjectName()).getLocation().toFile(),
-						this.preferenceStore.getString(PreferenceConstants.RELATIVE_CONFIG_PATH)));
-			}
-		} else {
-			this.setConfigFile(new File(this.preferenceStore.getString(PreferenceConstants.ABSOLUTE_CONFIG_PATH)));
-
-		}
+		this.setConfigFile(new File(this.preferenceStore.getString(PreferenceConstants.ABSOLUTE_CONFIG_PATH)));
```

Comment:
Never used the relative config; just curious why was it removed?

Reply:
We do not use the feature and as we have no use case for it currently it has been removed . It was an old idea .. maybe we'll add it somewhere in the future again